src/app/base/components/dropdown-iterative/dropdown-iterative.component.ts

```diff
@@ -0,0 +1,93 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { AbstractControl, FormGroup } from '@angular/forms';
+
+@Component({
+  selector: 'kern-dropdown-iterative',
+  templateUrl: './dropdown-iterative.component.html',
+  styleUrls: ['./dropdown-iterative.component.scss']
+})
+export class DropdownIterativeComponent implements OnInit {
+
+  @Input() arrayOptions = [];
+  @Input() condition;
+  @Input() valueIfConditionTrue;
+  @Input() valueIfConditionFalse;
+  @Input() buttonTooltip: string;
+  @Input() labelingTasks;
+  @Input() property: string;
```

Comment:
not really descriptive. i wouldn't know what property this is or what this is for

Reply:
I will remove both dropdown and dropdown iterative components and then replace dropdown-it with dropdown.